Dockerfile

```diff
@@ -33,7 +33,7 @@ FROM eclipse-temurin:17-jre-jammy as production
 # Create liquibase user
 RUN addgroup --gid 1001 liquibase && \
     adduser --disabled-password --uid 1001 --ingroup liquibase liquibase && \
-    mkdir /liquibase && chown liquibase /liquibase
+    mkdir /liquibase && chown root:root /liquibase
```

Comment:
This is a pretty big change from our previous set up. Will this cause issues with switching to the liquibase user later on in the dockerfile? It's even in the [best practice guide](https://docs.docker.com/develop/develop-images/instructions/#user) to use a non root user (which we still do), but if the root user owns the liquibase dir and tries to run it as the liquibase user will this cause issues?

Reply:
I think if the commands run, this is working. I think we still follow best practices because we do switch to a non root user. I was concerned about ownership of this directory being an issue, but that doesn't seem to be the case.